.github/workflows/build-images.yml

```diff
@@ -35,7 +35,8 @@ jobs:
       - uses: actions/checkout@v1
       - name: Build image
         run: |
-          ansible-galaxy install willshersystems.sshd
+          ansible-galaxy install willshersystems.sshd 
+          ansible-galaxy install community.general.npm
```

Comment:
[The docs](https://docs.ansible.com/ansible/latest/collections/community/general/npm_module.html) suggest including `collection` in this command, unless we're working from a different version.

Reply:
I could be wrong on the syntax, but I believe this would work to install JUST the `.npm` module from the `community.general`. If that's not accurate then yes, this should be swapped to `collection install` and just `community.general` with no `.npm`